kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -217,6 +227,36 @@
                         ++orderInGroup,
                         Width.LONG,
                         "Line separator ByteArrayFormat");
+
+      CONFIG_DEF.define(S3_PROXY_URL_CONFIG,
+                        Type.STRING,
+                        S3_PROXY_URL_DEFAULT,
+                        Importance.LOW,
+                        "S3 Proxy settings encoded in URL syntax",
+                        group,
+                        ++orderInGroup,
+                        Width.LONG,
+                        "S3 Proxy Settings");
+
+      CONFIG_DEF.define(S3_PROXY_USER_CONFIG,
+                        Type.PASSWORD,
```

Comment:
Should this be `Type.STRING` for the user?

Reply:
I felt that the username is also sensitive information to dump to the logs (same as the password). But that's up for discussion. /cc @ewencp

Reply:
Usernames that require authentication are not usually sensitive, whereas knowing having the username in the logs helps diagnose authentication/authorization issues.